.github/workflows/updatecli.yaml

```diff
@@ -27,12 +29,14 @@ jobs:
       - name: Install Updatecli
         uses: updatecli/updatecli-action@v2
         with:
-          version: v0.52.0
+          version: v0.54.0
       - run: updatecli apply --experimental
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
       - name: Git Auto Commit
         uses: stefanzweifel/git-auto-commit-action@v4.16.0
         with:
           commit_message: |
-            🛠 Updatecli pipeline bump #${{ github.event.pull_request.number }}
+            🛠 Updatecli pipeline bump
+          branch: ${{ github.event_name == 'workflow_dispatch' && 'updatecli-bump' || '' }}
```

Comment:
I think using an empty string for an input value does not trigger the default in the called action. Can you check that?

Reply:
looks good, I tested previously adding synchronize event here - commits are not here anymore because I've throw them away while rebasing